### PR TITLE
docs: Update readme with explore traces info

### PR DESCRIPTION
**Pull request by Rperry2174**
**What this PR does**:
This PR introduces a feature enhancement that integrates the Explore Traces app into the Tempo documentation to guide users on visualizing tracing data through a user-friendly, queryless interface.

**Checklist**
- [x] Tests updated
- [x] Documentation added
- [x] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,15 +8,31 @@
   <a href="https://goreportcard.com/report/github.com/grafana/tempo"><img src="https://goreportcard.com/badge/github.com/grafana/tempo" alt="Go Report Card" /></a>
 </p>
 
-Grafana Tempo is an open source, easy-to-use and high-scale distributed tracing backend. Tempo is cost-efficient, requiring only object storage to operate, and is deeply integrated with Grafana, Prometheus, and Loki.
+Grafana Tempo is an open source, easy-to-use, and high-scale distributed tracing backend. Tempo is cost-efficient, requiring only object storage to operate, and is deeply integrated with Grafana, Prometheus, and Loki. It is powered by [TraceQL](https://grafana.com/docs/tempo/latest/traceql/), a traces-first query language inspired by LogQL and PromQL which enables targeted queries or rich UI-driven analyses.
```
Comment (knylander-grafana) [inline README.md:11]: Is it correct to say that Tempo is powered by TraceQL? Rather, TraceQL is a language that lets you query the data held by the Tempo database.

File: README.md [hunk 2]
```diff
@@ -8,15 +8,31 @@
...
+- **Intuitive Trace Analysis**: Spot slow or error-prone traces with easy, point-and-click interactions.
+- **RED Metrics Overview**: Use Rate, Errors, and Duration metrics to highlight performance issues.
+- **Automated Comparison**: Identify problematic attributes with automatic trace comparison.
+- **Simplified Visualizations**: Access rich visual data without needing to construct TraceQL queries.
+
+![image](https://github.com/user-attachments/assets/991205df-1b27-489f-8ef0-1a05ee158996)
+
+To learn more see the following links:
+- [Explore Traces repo](https://github.com/grafana/explore-traces)
+- [Explore Traces Documentation](https://grafana.com/docs/grafana/latest/explore/simplified-exploration/traces/)
```
Comment (knylander-grafana) [inline README.md:31]:
```suggestion
- [Explore Traces documentation](https://grafana.com/docs/grafana/latest/explore/simplified-exploration/traces/)
```

**Review by knylander-grafana — APPROVED** [review]
Thank you for updating the docs! I have one question regarding the statement that you added about TraceQL.

File: README.md [hunk 3]
```diff
@@ -8,15 +8,31 @@
...
 </p>
 
-Grafana Tempo is an open source, easy-to-use and high-scale distributed tracing backend. Tempo is cost-efficient, requiring only object storage to operate, and is deeply integrated with Grafana, Prometheus, and Loki.
+Grafana Tempo is an open source, easy-to-use, and high-scale distributed tracing backend. Tempo is cost-efficient, requiring only object storage to operate, and is deeply integrated with Grafana, Prometheus, and Loki. It is powered by [TraceQL](https://grafana.com/docs/tempo/latest/traceql/), a traces-first query language inspired by LogQL and PromQL which enables targeted queries or rich UI-driven analyses.
 
-Tempo is Jaeger, Zipkin, Kafka, OpenCensus and OpenTelemetry compatible.  It ingests batches in any of the mentioned formats, buffers them and then writes them to Azure, GCS, S3 or local disk. As such, it is robust, cheap and easy to operate!
+Tempo is Jaeger, Zipkin, Kafka, OpenCensus, and OpenTelemetry compatible. It ingests batches in any of the mentioned formats, buffers them, and then writes them to Azure, GCS, S3, or local disk. As such, it is robust, cheap, and easy to operate!
```
Comment (knylander-grafana) [inline README.md:13]: I moved this sentence to the above area. 
```suggestion
```

File: README.md (hunk 1 above)
Comment (knylander-grafana) [reply README.md:11]: what about this? 
```suggestion
Grafana Tempo is an open source, easy-to-use, and high-scale distributed tracing backend. Tempo is cost-efficient, requiring only object storage to operate, and is deeply integrated with Grafana, Prometheus, and Loki. 

Tempo is Jaeger, Zipkin, Kafka, OpenCensus, and OpenTelemetry compatible. It ingests batches in any of the mentioned formats, buffers them, and then writes them to Azure, GCS, S3, or local disk. As such, it is robust, cheap, and easy to operate!

## TraceQL

Tempo provides [TraceQL](https://grafana.com/docs/tempo/latest/traceql/), a traces-first query language inspired by LogQL and PromQL, which enables targeted queries or rich UI-driven analyses. 

### TraceQL metrics 

[TraceQL metrics](https://grafana.com/docs/tempo/latest/traceql/metrics-queries/) is an experimental feature in Grafana Tempo that creates metrics from traces. Metric queries extend trace queries by applying a function to trace query results. This powerful feature allows for ad hoc aggregation of any existing TraceQL query by any dimension available in your traces, much in the same way that LogQL metric queries create metrics from logs. 
```

File: README.md [hunk 4]
```diff
@@ -8,15 +8,31 @@
...
 
-Tempo implements [TraceQL](https://grafana.com/docs/tempo/latest/traceql/), a traces-first query language inspired by LogQL and PromQL. This query language allows users to very precisely and easily select spans and jump directly to the spans fulfilling the specified conditions:
+## Business Value of Distributed Tracing
+Distributed tracing helps teams quickly pinpoint performance issues and understand the flow of requests across services. The Explore Traces UI simplifies this process by offering a user-friendly interface to view and analyze trace data, making it easier to identify and resolve issues without needing to write complex queries.
```
Comment (knylander-grafana) [inline README.md:16]:
```suggestion
## Business value of distributed tracing

Distributed tracing helps teams quickly pinpoint performance issues and understand the flow of requests across services. The Explore Traces UI simplifies this process by offering a user-friendly interface to view and analyze trace data, making it easier to identify and resolve issues without needing to write complex queries.

Refer to [use traces to find solutions](https://grafana.com/docs/tempo/latest/introduction/solutions-with-traces/)t o learn more about how you can use distributed tracing to investigate and solve issues. 

```

**Comment by knylander-grafana** [comment]
@Rperry2174 have you signed the CLA?

**Comment by derek-cadzow** [comment]
@knylander-grafana 
Kim, since this already being looked at, can you upgrade the doc status from Needs Triage to the appropriate current status?

**Pull request merged**